src/client/components/document-filler.js

```diff
@@ -1,100 +0,0 @@
-const React = require('react');
-const h = require('react-hyperscript');
-const { Link } = require('react-router-dom');
```

Comment:
Can you please un-delete this file.  It will cause more merge conflicts down the road.